lib/stores/http.ts

```diff
@@ -11,11 +11,15 @@ export const useHttpStore = defineStore('http', {
         ...options,
       })
 
+      if (!res.ok) {
+        throw new Error(await res.text())
+      }
+
```

Comment:
This is to handle the errors that come through as 'plain/text', I'm not sure ALL the API errors come through this way, so it might warrant some more digging